addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -107,6 +107,8 @@ def write(self, values):
             raise AccessError(_("You cannot access timesheets that are not yours."))
 
         values = self._timesheet_preprocess(values)
+        if 'name' in values and not values.get('name'):
+            values['name'] = _('/')
```

Comment:
Not sure if it's necessary to translate '/'

Reply:
Yep, not really useful.

Reply:
@tivisse , done Thanks